mlshvch/hometask-5/9.sql

```diff
@@ -0,0 +1,9 @@
+SELECT CONCAT(a.first_name, ' ', a.last_name) AS name, FA.count
+FROM actor AS a
+         LEFT JOIN
+     (SELECT fa.actor_id, COUNT(fa.actor_id) as count
+      FROM film_actor AS fa
+      GROUP BY fa.actor_id
+         ) AS FA
```

Comment:
You don't need a subquery here, you can get away with JOIN, HAVING and GROUP BY.